homeassistant/components/lookin/media_player.py

```diff
@@ -82,7 +82,6 @@ def __init__(
     ) -> None:
         """Init the lookin media player."""
         self._attr_device_class = device_class
-        self._attr_supported_features: int = 0
```

Comment:
Just to make sure, this was removed because it is not used by this media player?

Reply:
It is the default :)

Reply:
As frenck said, it's the default so there is no need to initialise it before calling `self._attr_supported_features |= feature`